README.md

```diff
@@ -45,7 +52,7 @@ For the protocol contracts addresses see https://docs.lido.fi/deployed-contracts
 * curl
 * cut
 * docker
```

Comment:
```suggestion
```

Reply:
there is the second mention of docker in the list also, but none is required anymore